### ISSUE TITLE
Warnings when MMS serving process starts on Python3

### ISSUE THREAD

**Issue by lupesko** [awslabs/mxnet-model-server#196]
When running MMS for serving on a Python3 environment, I get these warning messages:
```
/Users/hag/anaconda/envs/test-mms-3-6/lib/python3.6/site-packages/urllib3/contrib/pyopenssl.py:46: DeprecationWarning: OpenSSL.rand is deprecated - you should use os.urandom instead
import OpenSSL.SSL
/Users/hag/anaconda/envs/test-mms-3-6/lib/python3.6/site-packages/mms/service_manager.py:14: DeprecationWarning: the imp module is deprecated in favour of importlib; see the module's documentation for alternative uses
import imp
```

**Comment by yuruofeifei** [comment]
The first warning is because urllib3 is using a deprecating library.
The second is because we are using a deprecating library.
It's not easy to fix the first one. And users can always pass -W ignore::DeprecationWarning to suppress the warnings.

**Comment by frankfliu** [comment]
Fixed in MMS 1.0, 1.0 no longer depends on urllib3
